miner/worker.go

```diff
@@ -545,6 +545,12 @@ func (miner *Miner) commitTransactions(env *environment, plainTxs, blobTxs *tran
 				log.Debug("adding tx would exceed block DA size limit",
 					"hash", ltx.Hash, "txda", ltx.DABytes, "blockda", blockDABytes, "dalimit", miner.config.MaxDABlockSize)
 				txs.Pop()
+				// If the number of remaining bytes is too few to hold even the minimum possible transaction size,
+				// then we can stop early.
+				daBytesAfter.Add(daBytesAfter, types.MinTransactionSize)
```

Comment:
Down below we set `blockDABytes = daBytesAfter`. Wouldn't we now add an _additional_ min tx size for each tx, systematically distorting the block da bytes counter?

Reply:
yes indeed, thanks for catching that.  fixed by creating a copy.